common/app/renderers/DotcomRenderingService.scala

```diff
@@ -240,6 +244,44 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     post(ws, json, Configuration.rendering.baseURL + "/AMPInteractive", page.metadata.cacheTime)
   }
 
+  // This is complete rubbish, but we need to send format through to
+  // DCR, but aren't aware of what the format is at this stage so we just use the default
+  // TODO: Either remove this from DCR or from here
```

Comment:
As the comment suggests, this isn't a good way to do things, but will allow us to merge, and test this route if it is something we would like to explore.

Reply:
I think we do actually want to keep format! we use it for setting those little balls to the Pillar colour
!
![image](https://user-images.githubusercontent.com/21217225/190213321-0778b235-3271-47d4-aac2-1b315712a775.png)

Reply:
Indeed we do - there's more discussion here about how we might do this [here](https://github.com/guardian/frontend/pull/25467#discussion_r964678757).

The main issue is - in these onwards endpoints, we have no context of what the article is, so have no idea what the format is. In the chat above, we suggest passing to these endpoints, but that feels like quite a stinky pattern.

Other options were to drop format while in the CS world, or, if/when the fastly image work comes about, we'll be able to add format on the client as it will be aware of it there.